src/dmd/backend/cgsched.c

```diff
@@ -2317,9 +2327,8 @@ int Schedule::insert(Cinfo *ci)
     ic = -1;
     imin = 0;
     for (i = tblmax; i >= 0; i--)
```

Comment:
You could have moved the declaration of `i` down here too (as you did with the others) :wink:

Reply:
no, it's used outside the loop